File: problems/big_sum/submissions/accepted/big_sum_accepted.java
import java.util.Scanner;

public class big_sum_accepted {
    public static void main(String[] args) {
        // INPUT
        Scanner sc = new Scanner(System.in);
        long n = sc.nextLong();

        // summation formula
        long s = n * (n + 1) / 2;

        // OUTPUT
        System.out.println(s);
    }
}

File: problems/big_sum/submissions/accepted/big_sum_accepted.c
#include <stdio.h>

int main(int argc, char const *argv[])
{
    long n;
    // INPUT
    scanf("%ld", &n);

    // summation formula
    long s = n * (n+1) / 2;
    
    // OUTPUT
    printf("%ld", s);
    return 0;
}
